src/tests/unit/tests/electron/platform/android/rule-information.test.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { RuleInformation } from '../../../../../../electron/platform/android/rule-information';
+
+describe('RuleInformation', () => {
+    test('RuleId works correctly', () => {
+        const ruleIds = ['abc', 'xyz', 'this should work'];
+        for (const ruleId of ruleIds) {
+            const ruleInformation = new RuleInformation(ruleId, 'b', () => 'c');
+            expect(ruleId === ruleInformation.ruleId);
+        }
+    });
+
+    test('RuleDescription works correctly', () => {
+        const ruleDescriptions = ['abc', 'xyz', 'this should work'];
+        for (const ruleDescription of ruleDescriptions) {
+            const ruleInformation = new RuleInformation('a', ruleDescription, () => 'c');
+            expect(ruleDescription === ruleInformation.ruleDescription);
+        }
+    });
+
+    test('HowToFix works correctly', () => {
+        const howToFixStrings = ['abc', 'xyz', 'this should work'];
+        for (const howToFixString of howToFixStrings) {
+            const ruleInformation = new RuleInformation('a', 'b', () => howToFixString);
```

Comment:
while passing functions as parameters to classes (or other functions) we usually make use of a mocked function to verify it's being call with the expected parameters and the right amount of times. In general, we use `typemoq` for this purpose. You can find an example in this file `src\tests\unit\tests\common\components\copy-issue-details-button.test.tsx` for the `onClickMock`

Reply:
In this case, I'm trying to ensure that calling the howToFix getter actually ends up calling the howToFixDelegate property. I intentionally don't want to mock it out. I'll see if it makes sense in the other test cases